### ISSUE TITLE
[1.16.5; 1.3.5.1124] infernal tear configuration

### ISSUE THREAD

**Issue by Knxghtmare** [P3pp3rF1y/Reliquary#629]
_i have a few questions about configuring the infernal tear in my personal modpack..._

**1. is the whitelist capable of differentiating between items with specific NBT data?**
    i have RelicEx which gives me relics i can equip, such as rings, necklaces, body pieces, etc (for curio slot placement) but theyre just named "relicex:head_relic", "relicex:ring_relic" etc, and the rarities of said relics (of which there are 7) are differentiated in the nbt data with " {Rareness: "uncommon"} ", or " {Rareness: "epic"} ". i just want to be able to have different xp gains from each rarity of relic
    
so if it is able to do so, would i be able to do it like this?

![image](https://github.com/P3pp3rF1y/Reliquary/assets/81049188/54c9b98c-2f4c-4c8d-af8c-54c21828adc8)

**2. am i able to break up the one long singular line on the actual whitelist in the config file into multiple lines?**
    so basically i just want to split the long line into multiple to make it drastically easier to read and add more items, because i can sort each item by mod and find items ive already added easier
    
so this is why the whitelist looks like normally:
    
![image](https://github.com/P3pp3rF1y/Reliquary/assets/81049188/0fd73b1e-d45a-4789-81f5-78d391490d5e)

i was wondering if it was possible to do something like this:

![image](https://github.com/P3pp3rF1y/Reliquary/assets/81049188/a74e0b34-344f-406d-a9e9-af1825bd7c37)

and so one so forth.

any help would be greatly appreciated, i understand that these are pretty minor things and i am playing on a bit of an older version at this point

**Comment by P3pp3rF1y** [comment]
nope nbt data isn't possible and honestly that seems like too much hassle for what it would be worth given matching and how partial it can be and similar kind of stuff.

And it seems that toml implementation requires the whole list to be on one line so nope it wouldn't be possible to break it on multiple lines.
